src/gui/components.py

```diff
@@ -26,23 +29,98 @@ def add_qrcode(text, y=QR_PADDING, scr=None, style=None, width=None):
     qr = QRCode(scr)
     qr.set_text(text)
     qr.set_size(width)
-    qr.set_text(text)
```

Comment:
What is the purpose of this hack? I see that QR gets squeezed a little. But i dont understand why it has to be called again when it is called inside qr.set_size()...